Small fixes

hugwijst opened the pull request:
Fix a typo and remove an unused function.

hugwijst committed:
Rename getClusteNodeByName to getClusterNodeByName.

hugwijst committed:
Remove unused function.

gousiosg commented:
Please keep updating this branch with more small fixes you might do while going through the code. I will merge it when you say you're done.

gousiosg commented:
Also, please use more descriptive commit messages :smiley:

hugwijst commented:
Do you want me to reword the commit messages? I don't think there will be similar fixes from me for now, so you can merge the changes if you want.

hugwijst committed:
Remove duplicate method in MailMessage.

The method pairs (getFileName,getFilename) and (setFileName,setFilename)
performed the same function. The latter of both pairs has been removed.
